x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/rules/logical/FoldNullTests.java

```diff
@@ -98,7 +98,7 @@ public void testBasicNullFolding() {
         assertNullLiteral(foldNull(new Pow(EMPTY, Literal.NULL, Literal.NULL)));
         assertNullLiteral(foldNull(new DateFormat(EMPTY, Literal.NULL, Literal.NULL, null)));
         assertNullLiteral(foldNull(new DateParse(EMPTY, Literal.NULL, Literal.NULL, NULL)));
-        assertNullLiteral(foldNull(new DateTrunc(EMPTY, Literal.NULL, Literal.NULL)));
+        assertNullLiteral(foldNull(new DateTrunc(EMPTY, Literal.NULL, Literal.NULL, null)));
```

Comment:
Should we keep this consistent?
```suggestion
        assertNullLiteral(foldNull(new DateTrunc(EMPTY, Literal.NULL, Literal.NULL, TEST_CFG)));
```